packages/image_picker/image_picker/ios/Classes/FLTImagePickerPlugin_Test.h

```diff
@@ -0,0 +1,16 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// This header is available in the Test module. Import via "@import image_picker.Test;"
+
+#import <image_picker/FLTImagePickerPlugin.h>
+
+/// Methods exposed for unit testing.
+@interface FLTImagePickerPlugin ()
+
+@property(copy, nonatomic) FlutterResult result;
+- (void)handleSavedPathList:(NSArray *)pathList;
+- (void)imagePickerControllerDidCancel:(UIImagePickerController *)picker;
```

Comment:
These should always have had declaration comments; could you add those while you are exposing them?

Reply:
(Also, a small style nit: blank line between each one, in addition to the comment.

Reply:
Added declaration comments to each of the methods. Note that I simply copied already existing documentation from Apple documentation on the `imagePickerControllerDidCancel` method.